grunt/tasks/jest.js

```diff
@@ -1,84 +1,17 @@
 'use strict';
 
-var async = require('async');
 var fs = require('fs');
-var glob = require('glob');
 var grunt = require('grunt');
 var path = require('path');
 
-var rootPath = path.resolve('.');
```

Comment:
I don't think that's true. We want to exclude some files for sure, eg https://github.com/facebook/react/tree/355c49065386cbe33c026ba573c88c4e459ea328/src/shared/vendor/third_party (that file is showing up in the coverage report for this build). I'm also seeing coverage results for scripts/jest/test-framework-setup.js in the report.